src/emulate.c

```diff
@@ -76,49 +92,86 @@ static void rv_exception_default_handler(riscv_t *rv)
  * When a hardware breakpoint is triggered or an exception like address
  * misalignment, access fault, or page fault occurs during an instruction
  * fetch, load, or store operation, mtval is updated with the virtual address
- * that caused the fault. In the case of an illegal instruction trap, mtval
- * might be updated with the first XLEN or ILEN bits of the offending
- * instruction. For all other traps, mtval is simply set to zero. However,
- * it is worth noting that a future standard could redefine how mtval is
- * handled for different types of traps.
+ * that caused the fault. In the case of an illegal instruction trap, mtval *
```

Comment:
Don't change the existing comments unless you have strong reasons.

Reply:
Shall we change `mtval` parameter identifier to `tval` since we handle both supervisor tval and machine tval in the same function?

Reply:
> Shall we change `mtval` parameter identifier to `tval` since we handle both supervisor tval and machine tval in the same function?

I think so. Make it informative.